### DISCUSSION TITLE
Tandoor: Ingress and Home Assitant cloud

### DISCUSSION THREAD

**Discussion by jeromegoo** [alexbelgium/hassio-addons#480] [Q&A]
Hi,
I was pleased to see that ingress was added to tandoor home assistant add-on, but I can't make it work.

I'm wondering if it is compatible with a home assistant cloud url ?

And what key should I set for "SECRET_KEY" ?

Currently, when trying to access tandoor, I get this first page:
![image](https://user-images.githubusercontent.com/43144578/189875098-fcccece3-5ed4-436a-b1cc-12158f7634e5.png)

And then, when I try to create user account, I get this error:
![image](https://user-images.githubusercontent.com/43144578/189875169-332faf52-1713-4d58-9b89-ce1fe6721369.png)

Thanks,
Jérôme

**Comment by alexbelgium** [comment]
Hi, Mmh, that's a new one... It doesn't recognize the nabu casa domain as a valid one... Did you add it to the list of acceptable hosts in the addon options?

**Reply by jeromegoo** [reply]
Hi Alex,
I added it in the ALLOWED_HOSTS, this way:
![image](https://user-images.githubusercontent.com/43144578/190085115-0c62496c-6868-4fe0-9cb4-45f4b62932c0.png)

I tried with or without "https://".

**Comment by alexbelgium** [comment]
Thanks, I've pushed a new version that solves the issue, no need to add the nabu casa url in the allowed_hosts it is automatic

**Reply by jeromegoo** [reply]
Hi Alex,

Thanks for the update. I've tried it, and I don't have the error any more, but still have problems...
I can log in, but then, I'm stucked in this "Space" page. Wherever I click, I stay on this page. I can't even log out. (The only thing working on this page, seems to be the group creation).

![image](https://user-images.githubusercontent.com/43144578/191037483-ea084cb6-2993-4c04-ac33-4a2d5279b3ab.png)

Any idea why this is happening ? Do you think it can still be a problem with nabu casa ? Do you need logs ?

In the configuration, I didn't set anything in SECRET_KEY. Can it be the problem ? What should I set in this SECRET_KEY ?

Thanks,
Jérôme

**Comment by alexbelgium** [comment]
Hi, thanks I'll have to test; indeed I can replicate...
